openmsupply/mobile
# #2233 program requisition use soh at the end date of the period

Fixes #2233

## Change summary

add new helper to item data type to get stock on date
use that helper when creating requisitions lines for program
fix logic error where wrong field was displayed as requisition line current stock

## Testing

Steps to reproduce or otherwise test the changes of this PR:

* Use IC data file
* Use store that has plenty of transactions

- [ ] Create historic program requisition (on mobile)
- [ ] On desktop check that current stock in mobile matches (log in to store, go to item -> ledger -> print -> stock movements

### Docs

- [ ] Document that program requisitions current stock is soh at the period end date
